test/specs/mainnet/01_TransferringFunds/Arbitrum.spec.js

```diff
@@ -29,6 +29,12 @@ let incorrectTxHash =
   "0x1a44fe659f1502ff49315722be9c69bd76653baf422bfa30669a3485a52d4446";
 let pastTxHash =
   "0x1a44fe659f15021f49315792be9c69bd76953baf422bfa306f9a3485a52d4445";
+let tokenAddress = "0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8";
```

Comment:
All of these have to be constants as well.

Reply:
Created a file for the constants and modified all the test cases.